components/builder-worker/src/config.rs

```diff
@@ -54,9 +53,7 @@ impl Default for Config {
         jobsrv.insert("heartbeat".to_string(), "5567".to_string());
         Config {
             auth_token: "".to_string(),
-            data_path: hab_core::fs::svc_data_path("hab-builder-worker")
-                .to_string_lossy()
-                .into_owned(),
+            data_path: "/tmp".to_string(),
```

Comment:
Why would this switch to /tmp? We're breaking out of the `/hab/svc/FOO` in a way that feels wrong. What's the intent?